### PR TITLE
OOP school library: add Person, Student and Teacher classes

### PR DESCRIPTION
**Pull request by naanmohammed**
For this milestone I :

- Created class Person.
- Created class Student that inherits from Person.
- Created class Teacher  that inherits from Person.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Person
+  attr_reader :id, :name, :age
+
```
Comment (mukhammadrizooff) [inline person.rb]: - [ ] [REQUIRED] Good job so far! The `:id` should only be able to be retrieved so only the Getter is defined, please replace the `:id` to `attr_reader` and replace others to `attr_accessor`🙏🏼 

```ruby
  attr_accessor :name, :age 
  attr_reader :id 
```

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @naanmohammed, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- No linter errors ✔️ 
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (naanmohammed) [reply person.rb]: Thank you @mukhammadrizooff !

**Review by Ibizugbe — APPROVED** [review]
Hi @naanmohammed 👋👋👋, 

StandingOvationGIF (9)


Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights✅✅✅

- [x] `Person` class have been updated with the right value for the `attr_accessor` and the `attr_reader`👍.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**